apps/blockchain/scripts/sync_from_file.ex

```diff
@@ -0,0 +1,84 @@
+require Logger
+
+defmodule SyncFromFile do
```

Comment:
I've been wondering, would it make sense to write these as mix tasks? Then we (and others) can get access to the docs more easily. Mix tasks have docs when you do, 

```
# uses @shortdoc
mix help 

# or

# users @moduledoc
mix help task_name
```

What do you think?